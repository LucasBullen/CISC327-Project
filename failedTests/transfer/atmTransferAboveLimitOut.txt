8,12c8,12
< Logged into atm mode.
< Enter first account number.
< Enter second account number.
< Enter amount to transfer.
< Limit exceeded. Transfer cancelled.
---
> Logged into agent mode.
> Enter number of account to transfer from.
> Enter number of account to transfer to.
> Enter amount (in cents) to transfer.
> Transfer above $1000.00 limit. Transfer cancelled.
