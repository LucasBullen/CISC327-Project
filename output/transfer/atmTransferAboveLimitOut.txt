Select mode: atm or agent.
Logged into agent mode.
Enter account number to deposit to.
Enter amount to deposit.
Deposit successful.
Logged out.
Select mode: atm or agent.
Logged into atm mode.
Enter first account number.
Enter second account number.
Enter amount to transfer.
Limit exceeded. Transfer cancelled.
Logged out.
