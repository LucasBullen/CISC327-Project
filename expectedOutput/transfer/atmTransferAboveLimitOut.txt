Select mode: atm or agent.
Logged into agent mode.
Enter number of account to deposit into.
Enter amount (in cents) to deposit.
Deposit successful.
Logged out.
Select mode: atm or agent.
Logged into agent mode.
Enter number of account to transfer from.
Enter number of account to transfer to.
Enter amount (in cents) to transfer.
Transfer above $1000.00 limit. Transfer cancelled.
Logged out.
